ui/app/styles/components/sidebar.scss

```diff
@@ -41,6 +41,26 @@ $subNavOffset: 49px;
     box-shadow: -5px 0 10px -5px rgb(0 0 0 / 20%);
     border-radius: 16px;
   }
+
+  &.evaluations-sidebar {
+    display: flex;
+    gap: 1.5rem;
+    flex-direction: column;
+
+    & > .evaluation-response {
+      display: grid;
+      min-height: 200px;
+      grid-template-rows: auto 1fr;
+      overflow: hidden;
```

Comment:
The parent evaluation response container has overflow: hidden deliberately: it contains both the header and the json viewer, the latter of which has auto overflow.